### PR TITLE
Create a website consisting of 3 pages: Home, Calculator, and Quote.

### PR DESCRIPTION
**Pull request by m05e5**
# Math magicians: full website

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- The general layout should match these wireframes:
- For every page, you should create a route.
- navigation links for all 3 routes are displayed on each page.
- Use React Router to make the routes work.

### CODE REVIEW

File: src/components/CalculatePage.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-trailing-spaces */
```
Comment (Dipeshtwis) [inline src/components/CalculatePage.js:2]: - [ ] I can see you have disabled the `eslint` most of the pages. Please try to fix them as they are included just to make your code standard.

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-trailing-spaces */
```
Comment (Dipeshtwis) [inline src/components/Home.js:2]: - [ ] Same here. I can see you have disabled the `eslint` most of the pages. Please try to fix them as they are included just to make your code standard.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @m05e5, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of ROuter, switch, and route.
:+1: There is a website consisting of 3 pages: Home, Calculator, and Quote.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/CalculatePage.js (hunk 1 above)
Comment (m05e5) [reply src/components/CalculatePage.js:2]: ok, thanks. I fixed them but for the linebreak, I tried researching but I don't know how to resolve that. Please can you suggest me something?

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable linebreak-style */
```
Comment (mikenath223) [inline src/components/Header.js:1]: - [ ] As requested in the previous review kindly remove this eslint disable line on all components. To fix it you can simply try running `npx eslint . --fix` if on a windows system, that usually resolves it on my end.

Comment (m05e5) [reply src/components/Header.js:1]: its not working i tried it

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @m05e5,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Header.js (hunk 3 above)
Comment (Dipeshtwis) [reply src/components/Header.js]: Well Addressed!!!

**Pull request merged**